packages/core/src/Script.ts

```diff
@@ -152,6 +156,21 @@ export class Script extends Component {
     if (this.onLateUpdate !== prototype.onLateUpdate) {
       componentsManager.addOnLateUpdateScript(this);
     }
+    // Input callback.
+    if (
+      this.onPointerDown !== prototype.onPointerDown ||
+      this.onPointerUp !== prototype.onPointerUp ||
+      this.onPointerClick !== prototype.onPointerClick ||
+      this.onPointerDrag !== prototype.onPointerDrag ||
+      this.onPointerEnter !== prototype.onPointerEnter ||
+      this.onPointerExit !== prototype.onPointerExit
+    ) {
+      if (!this._listenInput) {
```

Comment:
Do not add `if(!this._listenInput)`， engine will ensure that the function is not called repeatedly.

Reply:
Done.